components/catalogs/Makefile

```
@@ -15,6 +15,7 @@ catalogs-tests:
 	$(MAKE) catalogs-lint-back
 	$(MAKE) catalogs-coupling-back
 	$(MAKE) catalogs-coverage-back
+	$(MAKE) catalogs-integration-back
```

```suggestion
	$(MAKE) catalogs-coverage-back
```

catalogs-coverage-back runs both unit & integration tests with coverage enabled, then merge both coverage and you can check the result by opening coverage/Catalogs/Back/Global/index.html in your browser.